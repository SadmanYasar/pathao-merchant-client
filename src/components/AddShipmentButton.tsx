import { Button } from '@chakra-ui/react'
import { motion } from 'framer-motion'

const AddShipmentButton = () => {
    return(
        <motion.div whileTap={{ scale: 0.9 }}>
            <Button
                size='lg'
                w='full'
                id='login-button'
                type='submit'
                bgColor={'red.400'}
                _hover={{
                    bgColor: 'red.500'
                
                }}
                _active={{
                    bgColor: 'red.500'
                }}>
                Add
            </Button>
        </motion.div>
    )
}

export default AddShipmentButton