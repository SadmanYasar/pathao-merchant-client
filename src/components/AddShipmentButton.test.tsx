import ReactTestRenderer from 'react-test-renderer'
import AddShipmentButton from './AddShipmentButton'

it('works', () => {
    const tree = ReactTestRenderer.create(<AddShipmentButton />).toJSON()
    expect(tree).toMatchSnapshot()
})