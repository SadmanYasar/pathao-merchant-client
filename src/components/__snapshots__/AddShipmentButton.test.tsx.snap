// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
<div
  style={Object {}}
>
  <button
    className="chakra-button css-1u4l63n"
    id="login-button"
    type="submit"
  >
    Add
  </button>
</div>
`;
